package.json

```diff
@@ -49,67 +49,69 @@
-    "@storybook/addon-actions": "^3.3.13",
-    "@storybook/addon-info": "^3.3.13",
-    "@storybook/addon-links": "^3.3.12",
-    "@storybook/addon-options": "^3.3.13",
-    "@storybook/addon-storysource": "^3.4.4",
-    "@storybook/addons": "^3.3.12",
-    "@storybook/react": "^3.3.13",
+    "@storybook/addon-actions": "^4.0.0-alpha.0",
+    "@storybook/addon-info": "^4.0.0-alpha.0",
+    "@storybook/addon-links": "^4.0.0-alpha.0",
+    "@storybook/addon-options": "^4.0.0-alpha.0",
+    "@storybook/addon-storysource": "^3.4.6",
+    "@storybook/addons": "^4.0.0-alpha.0",
+    "@storybook/react": "^4.0.0-alpha.0",
     "autobind-decorator": "^2.1.0",
-    "babel-core": "^6.26.0",
+    "babel-core": "^6.26.3",
     "babel-eslint": "^8.2.1",
-    "babel-jest": "^22.2.0",
+    "babel-jest": "^23.0.1",
     "babel-loader": "^7.1.2",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
+    "babel-plugin-transform-decorators-legacy": "^1.3.5",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
+    "babel-preset-env": "^1.7.0",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
-    "bootstrap": "^4.0.0",
+    "bootstrap": "^4.1.1",
     "copy-webpack-plugin": "^4.3.1",
     "coveralls": "^3.0.0",
     "css-loader": "^0.28.9",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "eslint": "^4.17.0",
     "eslint-config-airbnb": "^16.1.0",
-    "eslint-loader": "^1.9.0",
-    "eslint-plugin-import": "^2.8.0",
+    "eslint-loader": "^2.0.0",
+    "eslint-plugin-import": "^2.12.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.6.1",
-    "extract-text-webpack-plugin": "^3.0.2",
+    "eslint-plugin-react": "^7.9.1",
     "file-loader": "^1.1.6",
-    "html-webpack-plugin": "^2.30.1",
+    "html-webpack-plugin": "^3.2.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^22.2.1",
-    "node-sass": "^4.7.2",
+    "jest": "^23.1.0",
+    "mini-css-extract-plugin": "^0.4.0",
+    "node-sass": "^4.9.0",
     "react-router-dom": "^4.2.2",
-    "react-test-renderer": "^16.2.0",
+    "react-test-renderer": "^16.4.0",
     "redux-mock-store": "^1.5.1",
     "resolve-url-loader": "^2.2.1",
-    "sass-loader": "^6.0.6",
-    "style-loader": "^0.20.1",
-    "url-loader": "^0.6.2",
-    "webpack": "^3.10.0",
-    "webpack-dev-server": "^2.11.1"
+    "sass-loader": "^7.0.2",
+    "style-loader": "^0.21.0",
+    "url-loader": "^1.0.1",
+    "webpack": "^4.10.2",
+    "webpack-cli": "^3.0.2",
+    "webpack-dev-server": "^3.1.4"
   },
   "dependencies": {
-    "final-form": "^4.6.1",
-    "patternfly-react": "^1.16.4",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
Was this one intentional?

Reply:
I tried it, but should be removed, good catch!